### ISSUE TITLE
Database initialization bug

### ISSUE THREAD

**Issue by RieksJ** [AmpersandTarski/ampersand#347]
Perhaps this issue is related to #345, which @Michiel-s might verify. 

When using Ampersand v3.5.0[branch_#228:de766cd] (or Ampersand v3.5.0[branch_#228:3032de8]) in the (large) UETP project, database initialization results in errors which it did not do when using Ampersand v3.4.0[branch_#228-890de97].

This issue is blocking, since it prevents me from using the fix for #335 which was/is blocking, and is resolved in Ampersand v3.5.0[branch_#228:3032de8].

Here is a screenshot from the beginning of the log:
<img width="580" alt="picture1" src="https://cloud.githubusercontent.com/assets/8522589/14038467/6457f7a8-f251-11e5-964b-28cbaac2675c.png">

And here is a screenshot showing that the project is rather large, and gives multiple errors:
<img width="580" alt="picture2" src="https://cloud.githubusercontent.com/assets/8522589/14038469/75df133a-f251-11e5-9db4-c97717e2d46f.png">

**Comment by RieksJ** [comment]
@Michiel-s and I have made an attempt to find out where this stuff comes from. It seems that the database is structured completele wrong - details have been communicated between @hanjoosten and @Michiel-s by phone. 

In an attempt to contribute to the solution, I have generated ampersand.exe from a number of commits to see where the error is introduced. I have seen the following 

- Ampersand v3.4.0[branch_#228-890de97] is ok - it does not produce the error. Then, there was a pull request that inserted the master and the fix for issue 318 into branch_#228, which introduced the error in that branch.
- commit [f2e1a0d] in branch 'master' (11 mrt) does not produce the error.
- commit [6f562b5] in branch 'fix_for_issue_318' (12 mrt) does produce the error.
